### ISSUE TITLE
Anamnesis running but its White..

### ISSUE THREAD

**Issue by Bonnie-codes** [imchillin/Anamnesis#1381]
**Describe the bug**
Running the program and admin, it pops up a "'Anamnesis has not been validated against this game version' warning popup. I click "ok" and then it's running, but it's just a blank standalone window that I'm unable to interact with. I decided to do the "shift" click on the window option to reset it, but that's also not working. everything is up to date: FFXIV, FFXIV plugin, and Anemnesis. I've even tried a couple a trouble shoots b

**To Reproduce**
1) Open as admin for Anamnesis.
2) It shows white screen that I'm unable to even click or interact with.
3) Shift + click on the white window does not work. The only way to 

**Repeatability**
How often does the bug occur:
It's been doing this a little after DT release. I was trying to use Ktsis, but was informed that Anamnesis has been updated/working again. So I installed it, and have been getting this white window every time.  

**Here is a visual video if the issue.**
https://www.dropbox.com/scl/fi/m6jwtaobyv4ykfd81ujq5/20240829-0047-04.3217936.mp4?rlkey=vnc1yvl3qq9ss8gf5nuxkybxl&st=46w6s2p7&dl=0

**Comment by Bonnie-codes** [comment]
> Please follow the guide [here](https://github.com/imchillin/Anamnesis/wiki/Troubleshooting) to find your log file and post it.

I just updated to the new version today that was released three days ago and the issue is still occurring the same way from the previous versions. Thank you for showing me how to do the log file too, because I had no idea how to search for it. 

[2024-09-03-20-58-08.txt](https://github.com/user-attachments/files/16859930/2024-09-03-20-58-08.txt)
[2024-09-03-20-56-44.txt](https://github.com/user-attachments/files/16859932/2024-09-03-20-56-44.txt)

**Comment by StoiaCode** [comment]
Update your .NET first things first, link to it is on the front page of this github

**Comment by Bonnie-codes** [comment]
> Update your .NET first things first, link to it is on the front page of this github

I went ahead and uninstall the other versions and kept runtime 6.0.25 and .NET 8.0.03), launched the Ana again, then a popped up a window that it required a completely different version than what was on the github page(?): 6.0.33
Installed it, and the results are still the same as before with the white screen.

So, I decided to uninstall the 8.0.3 version, ran Ana, same outcome of a white screen.
Installed .NET 8.0.4, it's now saying it requires the 6.0.0 version. So I installed it, launched Ana, same outcome of a white screen.

There was so many test runs with this, that I decided to do a complete wipe of all the version entirely to install the latest version which includes the following:
    • .NET SDK 8.0.401
    • .NET Runtime 8.0.8
    • ASP.NET Core Runtime 8.0.8
    • .NET Windows Desktop Runtime 8.0.8

Restarted PC and then...a new issue occurred: 

You must install or update .NET to run this application (Armoury Crate) trying to get me to downgrade to 6.0.0.

Found the only solution to keep AC from spazzing out about this issue is to uninstall and reinstall because apparently there was something wrong with the newyear's update going into 2024 from Asus.

Upon fixing that issue, it downgraded my windows runtime (v6.0.15), so I upgraded it to the required version on Ana's github page Windows Runtime (v6.0.25) and .NET is updated to (v8.0.401).  Restarted my PC. 

Right now I  ONLY have the following:
.NET is updated (v8.0.401)
Microsoft Windows Desktop Runtime (v6.0.33) x64 
Microsoft Windows Desktop Runtime (v6.0.25) x64

After ALL of this hassle.... launched Ana, and getting the same results of a white screen I cant interact with. It's running, but I cant click onto it for anything. The only thing I can click onto is the preview window to "close" the app.

[2024-09-04-16-04-57.txt](https://github.com/user-attachments/files/16881371/2024-09-04-16-04-57.txt)

I've even have Ana whitelisted on my security. And for this next log, I went ahead and just turned it off to see if maybe that was the issue. Still giving me the same white screen issue. 

[2024-09-04-16-37-48.txt](https://github.com/user-attachments/files/16881375/2024-09-04-16-37-48.txt)

**Comment by StoiaCode** [comment]
What an endevour, but yeah it should have been just fine if you just installed the one linked and ignored everything else. I can see that at least worked though, you are running ana with the 6.0.33 version which should work. 

I can see a hint that the working Directory is C:\Anamnesis, is this correct, or is your Ana in some other folder? For sanity you can try to move the file somewhere else, another drive or something preferably like that if you have? Just for testing even the Desktop maybe. 

I cant find anything else that appears weird in the logs, you are on Global Servers as well so no Korean shenanigans...

**Comment by Bonnie-codes** [comment]
> What an endevour, but yeah it should have been just fine if you just installed the one linked and ignored everything else. I can see that at least worked though, you are running ana with the 6.0.33 version which should work.
> 
> I can see a hint that the working Directory is C:\Anamnesis, is this correct, or is your Ana in some other folder? For sanity you can try to move the file somewhere else, another drive or something preferably like that if you have? Just for testing even the Desktop maybe.
> 
> I cant find anything else that appears weird in the logs, you are on Global Servers as well so no Korean shenanigans...

So sorry for the late response! I will move it as soon as I get off work to se if it will work that way. 

I am curious, wouldn’t “OneDrive” cause an issue as well? Because I’m tempted to turn it off since I already manually backup my important documents and files.

**Comment by Bonnie-codes** [comment]
> What an endevour, but yeah it should have been just fine if you just installed the one linked and ignored everything else. I can see that at least worked though, you are running ana with the 6.0.33 version which should work.
> 
> I can see a hint that the working Directory is C:\Anamnesis, is this correct, or is your Ana in some other folder? For sanity you can try to move the file somewhere else, another drive or something preferably like that if you have? Just for testing even the Desktop maybe.
> 
> I cant find anything else that appears weird in the logs, you are on Global Servers as well so no Korean shenanigans...

I extracted the original folder for Anamnesis into the C drive and even run the program as Admin ("C:\Anamnesis\Anamnesis.exe").  But I went ahead and took your advice and moved the program out of the folder to Desktop:
[2024-09-10-22-18-26.txt](https://github.com/user-attachments/files/16957555/2024-09-10-22-18-26.txt)

Closed ana and rebooted it without admin from desktop: 
[2024-09-10-22-22-49.txt](https://github.com/user-attachments/files/16957558/2024-09-10-22-22-49.txt)

Closed ana and rebooted it with admin from desktop:
[2024-09-10-22-24-49.txt](https://github.com/user-attachments/files/16957561/2024-09-10-22-24-49.txt)

Closed Ana and moved it to C drive ("C:\Anamnesis.exe") and started it up as well... I think I'm running out of options on places this to be moved, but while I was doing all of the tests logs, I had already turned my onedrive off, because I thought that might of been the cause of the problem since it's been trying to constantly sync. But here's the last log: 
[2024-09-10-22-31-33.txt](https://github.com/user-attachments/files/16957567/2024-09-10-22-31-33.txt)

**Comment by Bonnie-codes** [comment]
I finally figured it out.. And I'm absolutely relieved that it's working. I noticed my firewall had **multiple** saved whitelisted Ana's inside of it from different paths and used the brand new path where I saved the official Ana in my C drive. I just completely deleted all of them, restarted my PC. And it didn't work at first, HOWEVER, I decided to even unplug both of my HDMI screens (since I have two monitors attached to my laptop). Plugged them back in and **there** it was!
